bitnami/mongodb/templates/replicaset/statefulset.yaml

```diff
@@ -215,7 +215,7 @@ spec:
                 fieldRef:
                   fieldPath: metadata.namespace
             - name: K8S_SERVICE_NAME
-              value: "{{ include "mongodb.fullname" . }}-headless"
+              value: "{{ include "mongodb.servicename" . }}"
```

Comment:
change servicename to servicenameOverride

Reply:
Fixed